### DISCUSSION TITLE
Does Perplexity train on my API requests even with Pro or enterprise Pro account?

### DISCUSSION THREAD

**Discussion by sungryull-sohn** [perplexityai/api-discussion#46] [Q&A]
On API FAQ, it clearly states that Perplexity uses my API request contents for training their AI.

https://docs.perplexity.ai/faq/faq#will-user-data-submitted-through-the-api-be-used-for-model-training-or-other-purposes

Is this also the case if I have Pro or enterprise Pro account?
Is there any plan/way to prevent Perplexity from using my API request contents for training?

**Comment by aarashy** [comment]
Hey, this was a typo in the FAQ and has been fixed. Perplexity API does not and has never collected the request/response content of API requests, they are not persistent. If this ever changes in the future we'd be sure to make it clear.
